README.md

```diff
@@ -71,7 +72,7 @@ val users :
 Finally to generate SQL from the query, one needs to define what exactly to
 select and how to parse each column:
 ```ocaml
-# let sql, parse_row = Ch_queries.query {%q|SELECT id FROM db.users|} Row.(fun __q -> int {%e|q.id|});;
+# let sql, parse_row = Ch_queries.query {%q|SELECT users.id FROM db.users|} Row.(fun __q -> int {%e|q.id|});;
```

Comment:
Maybe some explicit explanation about not using unqualified names would be useful ?